src/components/ControlBar/ViewControlBar/styles/DragHandle.module.css

```diff
@@ -0,0 +1,15 @@
+.draghandle {
+    position: absolute;
+    bottom: 0px;
```

Comment:
```suggestion
    bottom: 0;
```
Same for the others below ⬇️

Reply:
Done